htmd/parameterization/cli.py

```diff
@@ -63,6 +64,25 @@ def getArgumentParser():
 
     return parser
 
+def _get_molecule(args):
```

Comment:
I think this is more than just get, since it will do some processing according to the fftype, for example

Reply:
My comment maintains

Reply:
`_process_molecule`

Reply:
Changed!